zh_tw/xml/ha_clvm.xml

```diff
@@ -306,7 +306,7 @@
     </step>
    </procedure>
    <para>
-    如需關於 LVM2 的更多資訊，請參閱《SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">12 SP5</phrase></phrase> <citetitle> 儲存管理指南</citetitle>》：<link xlink:href="dsc;/sles-12/html/SLES-all/cha-lvm.html"/>。
+    如需關於 LVM2 的更多資訊，請參閱《SUSE Linux Enterprise Server <phrase role="productnumber"><phrase os="sles">12 SP5</phrase></phrase> <citetitle> 儲存管理指南</citetitle>》：<link xlink:href="dsc;/sle-ha/12-SP5/single-html/SLE-HA-guide/#book-sleha"/>。
```

Comment:
Please revert this change - see https://github.com/SUSE/doc-sleha/pull/199#discussion_r844098808 for reason why.